src/mesher/Mesher.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+#include "gmsh.h"
+#include <string>
+#include "TopoDS_Shape.hxx"
+
+
+class Mesher 
```

Comment:
Can you put this class in the `tigl` namespace please?

Reply:
Yes I'll do that